### DISCUSSION TITLE
airflow webserver fail to start

### DISCUSSION THREAD

**Discussion by juhonkang** [apache/airflow#23478] [General]
### Apache Airflow version

2.3.0 (latest released)

### What happened

ERROR: The `secret_key` setting under the webserver config has an insecure value - Airflow has failed safe and refuses to start. Please change this value to a new, per-environment, randomly generated 
string, for example using this command `openssl rand -hex 30`

### Operating System

Ubuntu 22.04

### Deployment

Other

### Deployment details

from conda forge channel of conda

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by tirkarthi** [comment]
This change was done in https://github.com/apache/airflow/pull/12747 . I guess you need to update your `secret_key` value as suggested in the error message. Is this a fresh installation or you are upgrading from another Airflow version?

**Comment by potiuk** [comment]
Yes. You MUST generate random value for secret - historicaly there was a default value which should be changed but some people did not do it - which left them open to anyone logging in to their server without authentication. Airflow 2 prevents from using such "known" value and requires you to generate it.

This is somethng you shoudl fix in your configuration (exactly as the error explains).
